cmake/config.cmake

```diff
@@ -172,6 +172,11 @@ set(USE_ROCBLAS OFF)
 # Whether use contrib sort
 set(USE_SORT ON)
 
+# Whether use contrib extern (use ";" to separate multiple externs)
+# Available externs:
+#   dnnl
+set(USE_EXTERN none)
```

Comment:
USE_EXTERN is a bit confusing, perhaps should be USE_DNNL_CODEGEN

Reply:
given there can be multiple "EXTERNs" it may not makes sense to use extern

Reply:
done